include/dlr_common.h

```diff
@@ -111,6 +115,23 @@ class DLRModel {
   virtual const char* GetBackend() const = 0;
   virtual void SetNumThreads(int threads) = 0;
   virtual void UseCPUAffinity(bool use) = 0;
+
+  virtual bool HasMetadata() const {
+    return false;
+  }
+
+  virtual const char* GetOutputName(const int index) const {
+    LOG(FATAL) << "GetOutputName is not supported yet!";
+  }
+
+  virtual int GetOutputIndex(const char* name) const {
+    LOG(FATAL) << "GetOutputIndex is not supported yet!";
+    return -1;
```

Comment:
Lets add comment `// unreachable`. `return -1 will never be executed, because LOG(FATAL) throws an exception.

Reply:
ok.

Reply:
This is a virtual function `// unreachable ` is misleading.